tests/unittest_prioq.c

```diff
@@ -1,10 +1,13 @@
 #include <check.h>
 
 #include <stdlib.h>
-
+#include <stdio.h>
 #include "prioq.h"
+#include "gen_allocdefs.h"
+
+GEN_ALLOC_readyplus(prioq,struct prioq_elt,p,len,a,100,check_readyplus)
```

Comment:
I find it difficult to understanding the pre-processor that convert a specific function name to *_readyplus, so I try to write tests for it to understand it better